packages/flags/src/index.ts

```diff
@@ -2,7 +2,8 @@ export type Flag =
   | 'PERSISTENT_EXAMPLE'
   | 'USER_PROFILE_EDIT_SKILLS'
   | 'USER_PROFILE_EDIT_WORKS'
-  | 'UPCOMING_EVENTS';
+  | 'UPCOMING_EVENTS'
+  | 'GROUP_UPCOMING_EVENTS';
```

Comment:
not sure it needs another flag, but okay. in that case I think we want to enable it straight away though? the other one is also already enabled